action/repo_add.go

```diff
@@ -90,7 +90,7 @@ var RepoAdd = &cli.Command{
 			Name:    "active",
 			Aliases: []string{"a"},
 			Usage:   "current status of the repository",
-			Value:   "false",
+			Value:   "true",
```

Comment:
the whole flag almost seems inconsequential for `add` because of https://github.com/go-vela/server/blob/master/api/repo.go#L99-L101 . should we just drop it all-together? or are there other changes planned?

Reply:
That looks more like a bug to me 🤷 

Instead I would expect those lines to look like this:

https://github.com/go-vela/server/blob/master/api/repo.go#L466-L469

Reply:
I'm good with that. Just calling it out.